amr-wind/turbulence/LES/Kosovic.cpp

```diff
@@ -25,7 +26,6 @@ Kosovic<Transport>::Kosovic(CFDSim& sim)
     m_Cs = std::sqrt(8 * (1 + m_Cb) / (27 * M_PI * M_PI));
     m_C1 = std::sqrt(960) * m_Cb / (7 * (1 + m_Cb) * m_Sk);
     m_C2 = m_C1;
-    pp.query("refMOL", m_refMOL);
```

Comment:
I like that you renamed this but I am worried about backwards compatability. Can you add an abort if someone tries to use this refMOL? There's a pp is present function you can use.